packaging/deps.sh

```diff
@@ -40,7 +40,8 @@ should_build() {
 
     if [[ -z "${DEPS_NONINTERACTIVE+x}" ]] && [[ -t 1 ]]; then
         echo
-        case "$(read -p "Do you wan't to install '${id}' into '${PREFIX}'? This is used ${use}. (y/N)")" in
+        read -p "Do you wan't to install '${id}' into '${PREFIX}'? This is used ${use}. (y/N) "
+        case "${REPLY}" in
```

Comment:
You should probably add a variable name to the read command as `REPLY` is parsed slightly differently than other variables.

See the bash `read` manpage: https://ss64.com/bash/read.html
> If no names are supplied, the line read is assigned to the variable $REPLY.

This means `REPLY` also includes all whitespace in the line, while an explicit name has the whitespace trimmed.

Reply:
Strange, the old approach didnt work for me.. My new one does. I also cant see that it appends a newline:

```bash
stv0g@GER-L-06044:/$ read
hallo
stv0g@GER-L-06044:/$ echo $REPLY
hallo
stv0g@GER-L-06044:/$ [[ $REPLY == "hallo" ]] && echo matches
matches
```